### ISSUE TITLE
octodns-dump failed with "unescaped ; in" for provider Selectel

### ISSUE THREAD

**Issue by antonydevanchi** [github/octodns#543]
**Command:** 

`octodns-dump --config=./config.yaml --output=./config domain.tld. selectel`

**Output:** 

`- unescaped ; in "v=DMARC1; p=none; sp=none; aspf=s; adkim=s; fo=1; pct=100; rf=afrf; ri=3600; rua=mailto:dmarc-reports@domain.tld; ruf=mailto:dmarc-reports@domain.tld"`

**Context:**

```
def _data_for_TXT(self, _type, records):
        return {
            'ttl': records[0]['ttl'],
            'type': _type,
            'values': [r['content'] for r in records],
        }
```

**Fix:**

Add:

```
def escape_semicolon(s):
    assert s
    return s.replace(';', '\\;')
```

Change:

`'values': [escape_semicolon(r['content']) for r in records]`

Tests passed, lint passed, coverage — 100%.
Please, make PR and merge it IDFK how it works, just first day in internet, you know.

**Comment by ross** [comment]
You might try the lenient flag if you haven't already: https://github.com/github/octodns/blob/32c05af6b9fc0d61dc184c5bc03bd114d044b10f/octodns/cmds/dump.py#L21-L23. That should allow you to dump things without validating them as strickly. You can then go through and edit the resulting yaml to fix the bit that's failing validation.

**Comment by antonydevanchi** [comment]
Fixed as #580
